packages/api/src/routes/sensorRoute.js

```diff
@@ -46,5 +46,5 @@ router.post('/reading/invalidate', SensorController.invalidateReadings);
 router.post('/unclaim', SensorController.retireSensor);
 router.get('/:location_id/reading_type', SensorController.getSensorReadingTypes);
 router.get('/partner/:partner_id/brand_name', SensorController.getBrandName);
-router.get('/reading/visualization', SensorController.getAllSensorReadingsByLocationIds);
+router.post('/reading/visualization', SensorController.getAllSensorReadingsByLocationIds);
```

Comment:
Is there a reason why this is being switched to a POST request? If I'm understanding what's going on, this endpoint retrieves the sensor readings, so it should be a GET request?

Reply:
it was POST initially, I am using the post method to send the data in the request body. I got this new change from database migration PR that I had to again change it from GET to POST in this PR.

Reply:
👍